interceptor.go

```diff
@@ -0,0 +1,54 @@
+// +build !js
+
+package webrtc
+
+import (
+	"io"
+
+	"github.com/pion/rtcp"
+	"github.com/pion/rtp"
+)
+
+type WriteRTP func(p *rtp.Packet, attributes map[interface{}]interface{}) (int, error)
+type ReadRTP func() (*rtp.Packet, map[interface{}]interface{}, error)
+type WriteRTCP func(pkts []rtcp.Packet, attributes map[interface{}]interface{}) (int, error)
+type ReadRTCP func() ([]rtcp.Packet, map[interface{}]interface{}, error)
+
+type Interceptor interface {
+	BindReadRTCP(read ReadRTCP) ReadRTCP     // TODO: call this
+	BindWriteRTCP(write WriteRTCP) WriteRTCP // TODO: call this
+
+	BindLocalTrack(ctx *TrackLocalContext, write WriteRTP) WriteRTP
+	UnbindLocalTrack(ctx *TrackLocalContext)
+
+	BindRemoteTrack(ctx *TrackRemoteContext, read ReadRTP) ReadRTP
+	UnbindRemoteTrack(ctx *TrackRemoteContext)
+
+	io.Closer
+}
+
+type DummyInterceptor struct{}
```

Comment:
I would re-name this `NoOpInterceptor` or something like that. Maybe we can move to a test and use it for some simple things?

Reply:
I made it public so it can be embedded in other Interceptors if you do not want to implement all methods.